nemo_skills/evaluation/evaluator/koifeval.py

```
@@ -0,0 +1,1448 @@
+    # Run evaluation using the internal evaluation_main function
+    evaluation_main(
+        input_data=str(jsonl_file), input_response_data=str(jsonl_file), output_dir=str(output_dir)
+    )
+
+    # fusing eval metrics back into the generation file
+    with open(jsonl_file, "rt", encoding="utf-8") as f:
+        samples = [json.loads(line) for line in f]
+
+    with open(output_dir / "eval_results_loose.jsonl", "rt", encoding="utf-8") as f:
+        eval_results = [json.loads(line) for line in f]
+    for sample, eval_result in zip(samples, eval_results):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)